### DISCUSSION TITLE
Claim Owner Permissions

### DISCUSSION THREAD

**Discussion by Lollipop56-UniverseMC** [GriefPrevention/GriefPrevention#1185] [Ideas]
Hey there,

I would like to request that a configuration option is added, that allows us to set permissions that only the onwer of the claim can use, for example we have a chest shops plugin. It would be nice to have the ability to give them shop.destory.others in their own claims for example but it would then stop them breaking peoples shops where there is no claim or in other peoples claims because they would not have the permission anywhere else but in their own claim.

Another example for this would be useful for donator perks to allow people to do custom things, I think this would be a great little configuration option.

**Comment by RoboMWM** [comment]
I made PrettySimpleShop for that above-mentioned purpose.

As for the rest, an addon could implement this.
